datahandler.rb

```diff
@@ -0,0 +1,49 @@
+require_relative './jsonHandler'
+require_relative './student'
```

Comment:
- [ ] Please fix this error that happens when the user launches the program.
![image](https://user-images.githubusercontent.com/62290545/181073291-e1d5cf02-4062-498b-9bc0-7f7b6abdfa4c.png)